mbway/src/main/java/com/adyen/checkout/mbway/internal/ui/DefaultMBWayDelegate.kt

```diff
@@ -177,7 +178,7 @@ internal class DefaultMBWayDelegate(
     }
 
     companion object {
-        private const val ISO_CODE_PORTUGAL = "PT"
+        internal const val ISO_CODE_PORTUGAL = "PT"
```

Comment:
I think this can also change back to private
```suggestion
        private const val ISO_CODE_PORTUGAL = "PT"
```